### PR TITLE
SSCS-10895 remove PO attending from override fields

### PR DESCRIPTION
**Pull request by nickhill111**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

https://tools.hmcts.net/jira/browse/SSCS-10895

Remove poToAttend from the override fields but still allow updateListingRequirements to change the po to attend

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by bensonlievoco — APPROVED** [review]

File: benefit/data/sheets/CaseTypeTab/CaseTabType-sandl-nonprod.json [hunk 1]
```diff
@@ -1,20 +1,24 @@
...
+    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_judge", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "amendReasons", "TabFieldDisplayOrder": 13, "FieldShowCondition": "[STATE]=\"readyToList\""},
     {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_clerk", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "hearingState", "TabFieldDisplayOrder": 8},
-    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_clerk", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "defaultOverrideFields", "TabFieldDisplayOrder": 9},
+    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_clerk", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "defaultListingValues", "TabFieldDisplayOrder": 9},
+    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_clerk", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "dwpIsOfficerAttending", "TabFieldDisplayOrder": 9},
```
Comment (TeaWithLucas) [inline benefit/data/sheets/CaseTypeTab/CaseTabType-sandl-nonprod.json:9]: Is dwpIsOfficerAttending only to show for clerk?

Comment (nickhill111) [reply benefit/data/sheets/CaseTypeTab/CaseTabType-sandl-nonprod.json:9]: no, it shows for registrar, judge and superuser also

**Review by TeaWithLucas — CHANGES_REQUESTED** [review]
Can you bump the versions number in https://github.com/hmcts/sscs-ccd-definitions/blob/master/benefit/VERSION.yaml and https://github.com/hmcts/sscs-ccd-definitions/blob/master/benefit/data/sheets/CaseType/CaseType.json

**Review by TeaWithLucas — APPROVED** [review]

**Review by bensonlievoco — DISMISSED** [review]

**Review by bensonlievoco — APPROVED** [review]

File: benefit/data/sheets/ComplexTypes/ComplexTypes.json [hunk 2]
```diff
@@ -483,6 +483,5 @@
...
   {"LiveFrom": "01/07/2022", "ID": "overrideFields", "ListElementCode": "hearingWindow", "FieldType": "hearingWindow", "ElementLabel": "Hearing Window", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "01/07/2022", "ID": "overrideFields", "ListElementCode": "autoList", "FieldType": "YesOrNo", "ElementLabel": "Should be Auto-listed", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "01/07/2022", "ID": "overrideFields", "ListElementCode": "hearingVenueEpimsIds", "FieldType": "Collection", "FieldTypeParameter": "hearingVenueEpimsId", "ElementLabel": "Hearing Venue Choices", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "01/07/2022", "ID": "overrideFields", "ListElementCode": "poToAttend", "FieldType": "YesOrNo", "ElementLabel": "Is PO Office Attending?", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "01/07/2022", "ID": "overrideFields", "ListElementCode": "hearingVenueEpimsIds", "FieldType": "Collection", "FieldTypeParameter": "hearingVenueEpimsId", "ElementLabel": "Hearing Venue Choices", "SecurityClassification": "PUBLIC"}
```
Comment (hgulv1) [inline benefit/data/sheets/ComplexTypes/ComplexTypes.json:486]: did you double check this prod file change with  del/sean?

Comment (nickhill111) [reply benefit/data/sheets/ComplexTypes/ComplexTypes.json:486]: Yes, poToAttend shouldnt be in production anyways

**Review by EliandroGV1 — DISMISSED** [review]

**Review by hgulv1 — APPROVED** [review]

**Review by EliandroGV1 — APPROVED** [review]

**Pull request merged**